src/routes/(app)/+layout.svelte

```diff
@@ -37,31 +30,14 @@
 				>
 			</div>
 			<div class="buttons">
-				<Button onclick={() => (screen = 'registration')}>{$_('signUp')}</Button>
-				<Button onclick={() => (screen = 'login')} variant="secondary"
-					>{$_('login')}<ArrowRight size={24} /></Button
-				>
+				<Button href="/signup">{$_('signUp')}</Button>
+				<Button href="/login" variant="secondary">{$_('login')}<ArrowRight size={24} /></Button>
```

Comment:
Please use the constants instead of hardcoded routes.
```suggestion
				<Button href={REGISTER}>{$_('signUp')}</Button>
				<Button href={LOGIN} variant="secondary">{$_('login')}<ArrowRight size={24} /></Button>
```